src/shared/Navigation/Nav.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <b-nav id="nav" :vertical="vertical || false" :class="styling" :align="align || ''">
```

Comment:
Just in case there will be multiple navbars in one page, make the `id` dynamic with `props`

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA